app/lib/experiments.coffee

```diff
@@ -9,7 +10,7 @@ AnalyticsLogger = require 'lib/analytics'
 Define the active experiment here by using a string which exists in http://experiments.zooniverse.org/active_experiments
 If no experiments should be running right now, set this to null, false or ""
 ###
-ACTIVE_EXPERIMENT = null
+ACTIVE_EXPERIMENT = "SerengetiBlanksExperiment1"
```

Comment:
Apologies for my ignorance, but isn't the goal of PlanOut to avoid referencing specific experiments from the front-end?

The way I thought it worked was the front-end sent a request upon page load, sending any and all data that PlanOut needs to assign them an experiments. Like so:

```
{ user: USER_ZOONIVERSE_ID, project: 'serengeti' }
```

PlanOut would handle assigning them to experiments as currently determined by what ones we have setup and running, and it would return an object by which the front-end could configure itself with. In this case, something like the following:

```
{ cohort: 20, source: 'blank' }
```

I might be misunderstanding, so would appreciate any clarify given here.